helpers.md

```diff
@@ -715,6 +717,28 @@ The `abort` function throws a HTTP exception which will be rendered by the excep
 You may also provide the exception's response text:
 
     abort(401, 'Unauthorized.');
+    
+<a name="method-abort-if"></a>
+#### `abort_if()` {#collection-method}
+
+The `abort_if` function is a syntax sugar for next expression:
+
+    if($condition) {
+    	abort(401);
```

Comment:
I can rewrite it to

The `abort_if` function is a syntax sugar. `abort_if($condition, 401)` will become to:
```
if($condition) {
    abort(401);
}
```

is it ok?